### PR TITLE
wi(ARCH-680): Deploy schema registry and document usage

### PR DESCRIPTION
**Pull request by fabballe**
**What is the problem this Pull Request is trying to solve?**
 
**What is the chosen solution to this problem?**
 
**Link to the JIRA issue**

 
**Please check if the Pull Request fulfills these requirements**
- [ ] The PR commit message follows our [guidelines](https://github.com/Talend/daikon/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features, coverage should be over 75% in the new code)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in JIRA), if any, are all linked or available in the Pull Request

 
**[ ] This Pull Request introduces a breaking change**

### CODE REVIEW

**Review by sgandon — COMMENTED** [review]
Woudl you mind adding a small readme somewhere to explain what is the purpose of the extension please ?

**Comment by fabballe** [comment]
> Woudl you mind adding a small readme somewhere to explain what is the purpose of the extension please ?

I added a README file with a link to our Talend kafka policy where all the information are written

File: daikon-schema/daikon-cloudevent-extension/README.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+= Daikon - Daikon cloud event extension
+v0.1
+:toc:
+
+== Introduction
+
+This module provide a CloudEvent extension which define a subset of attributes to complete an event according to a Talend ecosystem.
```
Comment (sgandon) [inline daikon-schema/daikon-cloudevent-extension/README.adoc:7]:
```suggestion
This module provide a CloudEvent extension which define a subset of attributes to complete an event according to a Talend ecosystem including tenant id and correlation id.
```

**Review by sgandon — APPROVED** [review]
LGTM

**Review by sgandon — APPROVED** [review]
LGTM

**Pull request merged**